src/hooks/useBridgesApi/useCreateBridgeApi.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  BridgeRequest,
+  BridgesApi,
+  Configuration,
+  InlineResponse202,
+} from "@openapi/generated";
+import { useState } from "react";
+
+export function useCreateBridgeApi(
+  accessToken: string,
+  basePath: string
+): {
+  createBridge: (bridgeRequest: BridgeRequest) => Promise<void>;
+  bridge?: InlineResponse202;
+  isLoading: boolean;
+  error: unknown;
+} {
+  const [bridge, setBridge] = useState<InlineResponse202>();
+  const [error, setError] = useState<unknown>();
+  const [isLoading, setIsLoading] = useState(true);
+
+  const createBridge = async (bridgeRequest: BridgeRequest): Promise<void> => {
+    const bridgeApi = new BridgesApi(
+      new Configuration({
+        accessToken,
+        basePath,
+      })
```

Comment:
I think it's possible to pass an async function to retrieve the token when setting up `Configuration`. In this way we could let the SDK client handle it instead of doing it by ourselves before calling these hooks. 

This means changing the `accessToken` prop of the hook to receive the `getToken` method we retrieve from the auth context.
Does it make sense?